docs/style-guide.md

```diff
@@ -210,9 +216,10 @@ As users are moved away from NOMIS, references to 'global' should be able to be
 {% accordionSection "H" %}
 
 ### help content (DPS only)
+
 When directing users to help and support, always use this content:
 
-**Get help with DPS**   
+**Get help with DPS**  
```

Comment:
Nice that Prettier has kept the line break, but should it be a heading 4?

Reply:
I think this needs to stay as-is.  Although this looks like a heading, its not really a heading in the context of this documents hierarchy - this is content we are directing others to use. Which also means we can't specify a heading level as we don't know their document.